views/privacy-policy.pug

```diff
@@ -16,9 +16,7 @@ block content
                             #wbars
                                 .iub_content.legal_pp
                                     .iub_header
-                                        h2
-                                            | Privacy Policy of 
-                                            strong www.bootstrapcdn.com
+                                        h2 Privacy Policy of www.bootstrapcdn.com
```

Comment:
@jdorfman ^^ I'll leave UI changes to you.

Reply:
There's no UI change; the header here already uses `font-weight: 700`